.buildkite/cache-builder.yml

```diff
@@ -14,7 +14,7 @@ common_params:
   # Common environment values to use with the `env` key.
   - &common_env
     # Be sure to also update the `.xcode-version` file when updating the Xcode image/version here
-    IMAGE_ID: xcode-14.2
+    IMAGE_ID: xcode-14.3.1
```

Comment:
Admittedly, this was unrelated with the Gutenberg setup.

I did run into some Xcode version compatibility issue during the long development of this integration, but I think it should no longer be a problem.

Having said that, I think we might as well use the latest toolchain, no?